docs/TEAMS.md

```diff
@@ -0,0 +1,202 @@
+The Organic Maps is led by contributors from across the community. Different functional domains may periodically establish, develop, or dissolve their own leadership structures, whether formal or informal.
+
+The file contains the functional project structure. The structure of GitHub teams must match the content of this file. Any changes to memberships must be performed via a PR to this file.
+
+- [Android Team](#android-team)
+- [iOS Team](#ios-team)
+- [Map Data Team](#map-data-team)
+- [Map Styles Team](#map-styles-team)
+- [DevOps Team](#devops-team)
+- [Design Team](#design-team)
+- [Website Team](#website-team)
+- [Growth Team](#growth-team)
+- [Translation Team](#translation-team)
+- [User Support Team](#user-support-team)
+- [Legal Team](#legal-team)
+
+## Android Team
+
+[@organicmaps/android](https://github.com/orgs/organicmaps/teams/android/members)
+
+**Scope**:
+
+Android application development.
+
+**Members:**
+
+- @AndrewShkrob
+- @biodranik
+- @kavikhalique
+- @map-per
+- @RedAuburn
+- @rtsisyk
+- @vng
+
+## iOS Team
+
+[@organicmaps/ios](https://github.com/orgs/organicmaps/teams/core/members)
+
+**Scope:**
+
+iOS application development.
+
+**Members:** 
+
+- @biodranik
+- @fabwu
+- @kirylkaveryn
+- @vng
+
+## Map Data Team
+
+[@organicmaps/data](https://github.com/orgs/organicmaps/teams/data/members)
+
+**Scope:**
+
+Map data preparation and generation.
+
+**Members:** 
+
+- @vng
+
+## Map Styles Team
+
+[@organicmaps/styles](https://github.com/orgs/organicmaps/teams/styles/members)
+
+**Scope:**
+
+Map Styles.
+
+**Members:** 
+
+- @pastk
+- @map-per 
+- @dvdmrtnz
+- @patepelo
+- @euf
+- @biodranik
+
+## DevOps Team
+
+[@organicmaps/devops](https://github.com/orgs/organicmaps/teams/devops/members)
+
+**Scope:**
+
+- Infrastructure management
+- CI/CD pipeline maintenance
+- Security and compliance
+- Backup and disaster recovery
+- Monitoring and alerting
+- Automation and orchestration
+
+**Members:** 
+
+- @rtsisyk
+
+## Design Team
+
+[@organicmaps/design](https://github.com/orgs/organicmaps/teams/design/members)
+
+**Responsibilities:**
+
+- UI/UX research
+- Wireframing and prototyping
+- Visual design
+- Interaction design
+- Accessibility considerations
+- Design system development
+- Collaboration with development and documentation teams
+
+**Members:**
+
+- @dvdmrtnz
+- @euf
+- @patepelo
+- @RedAuburn
+- @rtsisyk
+
+
+## Website Team
+
+[@organicmaps/website](https://github.com/orgs/organicmaps/teams/website/members)
+
+**Scope**:
+
+- HTML, CSS, and JavaScript development of web pages
+
+**Members:**
+
+- @rtsisyk
+
+## Growth Team
+
+[@organicmaps/growth](https://github.com/orgs/organicmaps/teams/growth/members)
+
+**Scope:**
+
+- User acquisition strategies to attract new users to the project.
+- Conversion rate optimization to improve user engagement and retention.
+- Retention campaigns to keep existing users engaged and active.
```

Comment:
Up to you, but it looks unnecessarily detailed - e.g. these 3 lines are almost the same, could be just 1.

Reply:
How about adding strategies of contributors' acquisition and retention to team's scope?

Reply:
It's too detailed. I will shrink to 3-5 items.

Reply:
Updated.